rotkehlchen/accounting/export/csv.py

```diff
@@ -331,3 +352,23 @@ def export(
             return False, str(e)
 
         return True, ''
+
+    def export_history(
+            self,
+            events: list['HistoryBaseEntry'],
+            directory: Path,
+    ) -> tuple[bool, str]:
+        if len(events) == 0:
```

Comment:
I would move this logic to the `rest.py` file. You don't need the class information for anything and the `to_csv`logic should by an attribute of the event class

Reply:
done